cli.js

```diff
@@ -67,6 +67,15 @@ Arguments:
       showVersion: true,
       logger: Logger,
       domDiff: this.options.domDiff,
+      middleware: [
+        function (req, res, next) {
+          if (/.*\.php$/.test(req.url)) {
```

Comment:
Right here

Reply:
You are right - I first thought cli.js is just use by ``npm run sample`` but now I see that this is also used by the exposed package bin.  

Removed the middleware from cli.js. Now ``index.php`` will be downloaded instead of displayed when running ``npm run sample`` but I can live with that because there exists a test case for that 👍